controllers/admin.js

```diff
@@ -86,7 +102,7 @@ exports.userAdmin = function (aReq, aRes, aNext) {
       });
     });
 
-    aRes.render('userAdmin', options);
+    aRes.render('userAdmin', options); // NOTE: Watchpoint
```

Comment:
@sizzlemctwizzle Cc: @Zren 

Do either one of you know where this physical view is? The docs for *express* [res.render](http://expressjs.com/4x/api.html#res.render) *(and related [app.render](http://expressjs.com/4x/api.html#app.render))* don't seem to cover this.